core/src/test/scala/pickling/run/anypickler-unpickler.scala

```diff
@@ -0,0 +1,60 @@
+package scala.pickling.runtime
+
+import org.scalatest.FunSuite
+import scala.pickling._
+
+import scala.pickling.json.JsonFormats
+import scala.pickling.pickler.AllPicklers
+import scala.pickling.json.JSONPickle
+import scala.pickling.internal.HybridRuntime
+
+abstract class PicklingLogic extends Ops with AllPicklers with JsonFormats
+
+object PicklingProtocol extends {
+  val ignoreMe = internal.replaceRuntime(new HybridRuntime)
+} with PicklingLogic {
+  implicit val so = static.StaticOnly
+}
+
+case class Box[T](value: T)
+
+class AnyPicklerUnpickler extends FunSuite {
+
+  import PicklingProtocol._
+
+  test("AnyPickler should work") {
+
+    val b = Box(List(1,2,3,4,5))
+    val containerPickler = implicitly[Pickler[List[Int]]]
+    val pickled = b.pickle.value
```

Comment:
Sorry for the delay of my reply. Yes, I realised that, but even registering the pickler I believe there's no possible way `AnyPickler` would work. The major problem is that `AnyPickler` deals with generic types that are erased as if they were `Any` and therefore the key that uses for looking up in the map is `scala.pickling.runtime.Box[scala.Any]` which does not match `scala.pickling.runtime.Box[scala.collection.immutable.List[scala.Int]]`.
  
Even if we register the tag with the `Any` keyword, the value mapped to that key would not work for other generic types of `Box`. The only way to solve this problem would be have access to the actual types behind `Any`, but again, the class name will only return the erased type of Box.
  
I've also tried other workarounds that give no success.